test/parallel/test-tls-passphrase.js

```diff
@@ -11,11 +11,17 @@ var tls = require('tls');
 var fs = require('fs');
 var path = require('path');
 
-var key = fs.readFileSync(path.join(common.fixturesDir, 'pass-key.pem'));
+var passKey = fs.readFileSync(path.join(common.fixturesDir, 'pass-key.pem'));
+var rawKey = fs.readFileSync(path.join(common.fixturesDir, 'raw-key.pem'));
 var cert = fs.readFileSync(path.join(common.fixturesDir, 'pass-cert.pem'));
 
+assert(Buffer.isBuffer(passKey));
+assert(Buffer.isBuffer(cert));
+assert.equal(typeof passKey.toString(), 'string');
```

Comment:
Nit: `assert.strictEqual()` instead of `assert.equal()` here and anywhere else it shows up